### PR TITLE
Rename file ble.h into nrf_ble.h.

### PR DESCRIPTION
**Pull request by pan-**
This change will mitigate issues when nrf51-sdk is compiled on mbed-classic.

### CODE REVIEW

**Comment by ciarmcom** [comment]
Automatic CI verification build not done, please verify manually.

**Comment by pan-** [comment]
@rgrover @LiyouZhou Could you review this please ?

File: source/nordic_sdk/components/ble/common/ble_date_time.h [hunk 1]
```diff
@@ -30,8 +30,8 @@
  *
  */
 
-/* Attention!
```
Comment (rgrover) [inline source/nordic_sdk/components/ble/common/ble_date_time.h:33]: this file has only white space diffs. We should remove it from the pull request. Let's avoid un-necessary changes.

File: source/nordic_sdk/components/ble/common/ble_sensor_location.h [hunk 2]
```diff
@@ -31,10 +31,10 @@
  */
```
Comment (rgrover) [inline source/nordic_sdk/components/ble/common/ble_sensor_location.h:32]: only white space diffs. remove this from the PR

**Comment by rgrover** [comment]
looks OK except for the white-space diffs which are avoidable.

File: source/nordic_sdk/components/ble/common/ble_date_time.h (hunk 1 above)
Comment (pan-) [inline source/nordic_sdk/components/ble/common/ble_date_time.h:33]: On the next line, you will see that I have replaced the character `’` by `'`. 
It was the cause of some issue during the synchronization with mercurial.

File: source/nordic_sdk/components/ble/common/ble_sensor_location.h (hunk 2 above)
Comment (pan-) [inline source/nordic_sdk/components/ble/common/ble_sensor_location.h:32]: Same as before `’` => `'`.

**Comment by LiyouZhou** [comment]
I think whitespace changes should not be rolled together into the same PR. Ideally we want to keep changes to nordic file to a minimum, hence no whitespace changes but I don't know to what extend it is true in this repo.

**Comment by pan-** [comment]
Thanks everyone,

Will change my editor settings and redo the commit.

**Comment by pan-** [comment]
@LiyouZhou  Hope it is better, let me know.

**Comment by LiyouZhou** [comment]
Look good.

**Pull request merged**